Update documentation for swiftly (#1905)
swiftlang/vscode-swift: userdocs/userdocs.docc/Articles/Topics/supported-toolchains.md

Debugging with `lldb-dap`  | 6.0
<doc:docc-live-preview>    | 6.2

## Toolchain Management

The Swift extension will automatically detect installations of the Swift toolchain in your environment. It will look for a `swift` binary available in `PATH` and, if one cannot be found, will prompt you to [install a toolchain from Swift.org](https://www.swift.org/install).

> If you have multiple Swift toolchains installed on your system then you can use the command `Swift: Select Toolchain...` to tell the extension which toolchain you would like to use. This will show you a list of all toolchains that VS Code was able to find on your system and let you switch between them.

Reviewer: ```suggestion
If you have multiple Swift toolchains installed on your system, use the command `Swift: Select Toolchain...` to tell the extension which toolchain to use. The command shows you a list of all the toolchains that VS Code found on your system and lets you switch between them.
```

@Video(
    source: "toolchain-selection.mp4",
    alt: "A video showing a VS Code window. The command palette is opened to run the 'Swift: Select Toolchain...' command which is then used to select Xcode as the preferred toolchain.",
    poster: "toolchain-selection.png"
)

## Swiftly Support

> The extension supports toolchains managed by [swiftly](https://github.com/swiftlang/swiftly), the Swift toolchain installer and manager. This is the recommended way of installing Swift toolchains on macOS and Linux. For instructions on installing swiftly see the [installation instructions on Swift.org](https://www.swift.org/install).

Author: I'm on the fence about this one. Swiftly's documentation always refers to itself as swiftly without the monospace and all lower-case (even when it's at the start of a sentence which is interesting). I also don't think that the monospaced font is super distinguishable between the standard font in the compiled documentation unless it's made bold as well.

> You can choose a swiftly managed toolchain to use from the `> Swift: Select Toolchain...` menu.

Reviewer: ```suggestion
Choose a swiftly managed toolchain to use from the `> Swift: Select Toolchain...` menu.
```
